docs/developers/documentation/index.md

```diff
@@ -0,0 +1,209 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.10.3
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+theme:
+  intro: This guide will walk you through submitting a new document to napari.org.
+---
```

Comment:
We probably don't actually need this here? We're not showing the intro anywhere and I don't think it gives us anything else...